### ISSUE TITLE
Toggle - Hear yourself (to check if it's working properly)

### ISSUE THREAD

**Issue by aha999** [lyrebird-voice-changer/lyrebird#91]
There should be a toggle where you could hear yourself with a changed voice, so you can hear what it sounds like and to hear if it's working at least.

Add a toggle to hear yourself.

**Comment by harrego** [comment]
Agreed - I like this idea. I'll keep this in mind unless someone else feels like adding. :)

**Comment by CamoCatX** [comment]
I am in agreement as well. A valuable feature.
